### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by stevekinney**
This is an attempt to provide slightly more helpful suggestions for code reviewers instead of just picking someone at random.

### CODE REVIEW

**Review by rossedfort — APPROVED** [review]
🎉

**Comment by robholland** [comment]
Can't we just give everything to @feedmeapples ? :)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,4 +2,20 @@
 # the repo. Unless a later match takes precedence,
 # @temporalio/frontend-engineering will be requested for
 # review when someone opens a pull request.
-*       @temporalio/frontend-engineering
+*                                       @temporalio/frontend-engineering
+/src/routes/                            @temporalio/frontend-engineering
+/src/lib/holocene/                      @rossedfort
+/src/lib/components/                    @rossedfort
+/server/						                    @feedmeapples @robholland
```
Comment (robholland) [inline .github/CODEOWNERS:9]: Indent looks off for these lines.

**Review by robholland — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,4 +2,20 @@
...
+*.go							                      @feedmeapples @robholland
+/src/types                              @stevekinney
+/src/lib/models                         @stevekinney
+/src/lib/services                       @stevekinney
```
Comment (Alex-Tideman) [inline .github/CODEOWNERS:13]: Add me to these ones too

File: .github/CODEOWNERS (hunk 1 above)
Comment (stevekinney) [reply .github/CODEOWNERS]: Tabs versus spaces. 🫠

File: .github/CODEOWNERS (hunk 2 above)
Comment (stevekinney) [reply .github/CODEOWNERS]: Done!

**Review by Alex-Tideman — APPROVED** [review]

**Pull request merged**